doc/user-guide/cli.md

```diff
@@ -98,6 +98,21 @@ It can also be combined with the `--from-existing` for re-initializing from an e
 ploomber-cloud init --from-existing --force
 ```
 
+## Ignore a file or directory
+
+You can choose to ignore any file or directory by adding a list in the `ignore` field in `ploomber-cloud.json`. Example:
+
```

Comment:
explain what effect `ignore` has (it'll not include those files in the uploaded zip file)

Reply:
[Added more detailed explanation](https://github.com/ploomber/doc/pull/256/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R101-R121)